docker-compose.yml

```diff
@@ -1,17 +1,20 @@
-web:
-  image: colinxfleming/dcaf_case_management
-  command: rails s -p 3000 -b 0.0.0.0
-  volumes:
-    - .:/usr/src/app
-  ports:
-    - "3000:3000"
-  links:
-    - db
-  environment:
-    mongohost: db
-  env_file:
-    - web-variables.env
-db:
-  image: mongo
-  ports:
-    - "27017:27017"
+version: '3.0'
```

Comment:
Copy that. Let's touch base on this tomorrow, I think we might be able to roll out a change after this weekend (I got this project into a Circle beta) that might make this work. 

I'm gonna hold on merging but let's make it clear that this is an available option for folks in the meantime.